### PR TITLE
Fix manifest having too many files and not substituting

### PR DESCRIPTION
**Pull request by eivindjahren**
Also makes substitution of <IENS> and <ITER> for all parameters and responses

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Make sure unit tests pass locally after every commit (`git rebase -i main
      --exec 'pytest tests/ert/unit_tests -n logical -m "not integration_test"'`)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

File: tests/ert/unit_tests/test_run_path_creation.py [hunk 1]
```diff
@@ -664,41 +677,8 @@ def test_that_runpaths_are_raised_when_invalid(run_path, expected_raise, msg):
         _ = ErtConfig.from_file("config.ert")
 
 
-@pytest.mark.usefixtures("use_tmpdir")
-@pytest.mark.parametrize(
-    "placeholder",
-    ["<ERTCASE>", "<ERT-CASE>"],
```
Comment (oyvindeide) [inline tests/ert/unit_tests/test_run_path_creation.py:670]: Will we still replace this in runpath?

Comment (eivindjahren) [reply tests/ert/unit_tests/test_run_path_creation.py]: Sorry, this was just erronously removed. Added it back in.

File: src/ert/config/field.py [hunk 2]
```diff
@@ -166,7 +167,9 @@ def write_to_runpath(
         self, run_path: Path, real_nr: int, ensemble: Ensemble
     ) -> None:
         t = time.perf_counter()
-        file_out = run_path.joinpath(self.output_file)
+        file_out = run_path.joinpath(
+            substitute_runpath_name(str(self.output_file), real_nr, ensemble.iteration)
```
Comment (sondreso) [inline src/ert/config/field.py:171]: why `str` here and not other places? 🤔

**Review by sondreso — APPROVED** [review]
Except for my small question, this LGTM! 🚢

File: src/ert/config/field.py (hunk 2 above)
Comment (eivindjahren) [reply src/ert/config/field.py:171]: This is the only class that uses `Path` for `self.output_file`

**Pull request merged**